common/app/views/fragments/amp/header.scala.html

```diff
@@ -3,12 +3,18 @@
 @import conf.Configuration
 @import common.{LinkTo, NewNavigation}
 
-<header class="header" role="banner">
+<header class="header"
+        role="banner">
     <div class="header__inner gs-container">
-        <a class="header__supporter-cta" data-link-name="amp : nav : supporter-cta"
-        href="@Configuration.id.membershipUrl/supporter?INTCMP=AMP_HEADER_GU_SUPPORTER">
-            Become a <span>Supporter</span>
-        </a>
+        <div class="header__support">
```

Comment:
I think this element is a `cta` (call to action) and isn't really a "support" of of the header. Maybe rename to: `header__cta`?